biogrid/apps/biogrid-mvp/src/app/pages/input-page/input-page.css

```diff
@@ -31,38 +36,237 @@ br {
   width: 56.25rem;
 }
 
+#intro {
+  background: #adadad;
```

Comment:
nit: this grey gives the site a bit more of a dated look. what about a lighter version of the teal background color you have?

Reply:
Thanks for the suggestion! It looks a little cooler now :)